servers/src/main/java/tachyon/master/file/journal/AddMountPointEntry.java

```diff
@@ -36,7 +37,9 @@
    * @param tachyonPath the Tachyon path
    * @param ufsPath the UFS path
    */
-  public AddMountPointEntry(TachyonURI tachyonPath, TachyonURI ufsPath) {
+  @JsonCreator
+  public AddMountPointEntry(@JsonProperty("tachyonPath") TachyonURI tachyonPath,
```

Comment:
It is deserialized because Jackson finds out a suitable constructor, i.e. a constructor with only a String parameter